README.md

```diff
@@ -19,6 +20,12 @@ Install CyFi with pip:
 pip install cyfi
 ```
 
+If you're on a Mac, you should install from conda-forge instead:
+
+```
+conda install -c conda-forge cyfi -y
```

Comment:
I don't like this `-y` here. Conda has decided to have default behavior that they ask you if you want to install it, and it feels like overstepping to default people around it.

Reply:
I was on the fence about this. This feels like the sort of thing that if you're not used to working in the terminal and a bunch of text goes by and then it says "y/n", it's not necessarily obvious that this is standard behavior and typing "y" is a totally sensible thing to do as part of the install process. I just don't want the default conda behavior where it pauses to make people think there is an error when everything is still going fine.